### PR TITLE
dotCMS/core#24578 DotFavoritePortlet-add-spinner-when-going-to-Edit-Mode

### LINKED ISSUE #24578: Listing Page - When you click a page to preview, we are going to need a spinner (or spinning cursor, something) or something that shows things are "happening"
### Parent Issue

https://github.com/dotCMS/core/issues/24326

### Task

Listing Page - When you click a page to preview, we are going to need a spinner (or spinning cursor, something) or something that shows things are "happening"

### Mock
image

Use spinner component from primeNG
https://primeng.org/progressspinner

### Steps
- Go to `Pages` portlet
- Click on a page to redirect to EDIT MODE

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
### Proposed Changes
Listing Page - When you click a page to preview, we are going to need a spinner (or spinning cursor, something) or something that shows things are "happening"

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
image

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages.component.scss [hunk 1]
```diff
@@ -13,3 +13,56 @@
...
+        stroke-dasharray: 89, 200;
+        stroke-dashoffset: -35px;
+    }
+    100% {
+        stroke-dasharray: 89, 200;
+        stroke-dashoffset: -124px;
+    }
+}
+
+@keyframes p-progress-spinner-color {
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages.component.scss:53]: Isn't this provided by primeng? if not... shouldn't be global?

Comment (alfredo-dotcms) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages.component.scss]: fixed

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages.component.scss [hunk 2]
```diff
@@ -13,3 +13,19 @@
...
+.dot-pages__spinner-overlay {
+    align-content: center;
+    background-color: $white;
+    display: flex;
+    flex-wrap: wrap;
+    height: calc(100% - $spacing-7);
+    justify-content: center;
+    position: absolute;
+    width: calc(100% - $spacing-8);
+}
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages.component.scss:31]: Please add a video on this loading indicator and check in with Melissa if it should look like this globally?

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages.component.scss (hunk 2 above)
Comment (alfredo-dotcms) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages.component.scss:31]: video uploaded
https://user-images.githubusercontent.com/37185433/231851175-4cf40668-db36-44a0-96da-af3b1b56945b.mov

Comment (alfredo-dotcms) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages.component.scss:31]: @fmontes after discussing with Melissa, the style was updated!

https://user-images.githubusercontent.com/37185433/231879476-5628cfb6-df5b-4ec2-8988-6ea7022bdfae.mov

**Pull request merged**